studio/src/structure/portfolio.js

```diff
@@ -0,0 +1,11 @@
+import S from '@sanity/desk-tool/structure-builder'
+
+const portfolio = S.listItem().title('Portfolio').child(
```

Comment:
this is a very simple "desk" item that adds the filtered resource list to the main menu in Sanity Studio